client/src/pages/SearchPage.js

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import VideoLists from "../components/VideoLists";
+const SearchPage = ({ allVideos, getAllVideos, order }) => {
+  const [search, setSearch] = useState("");
+  const [videoSearch, setVideoSearch] = useState([]);
+  const [selectedVideo, setSelectedVideo] = useState([]);
+  const [selectedValue, setSelectedValue] = useState("");
+
+  const searchHandle = (e) => {
+    const searchText = e.target.value;
+    if (searchText === "") {
+      setSearch("");
+      setVideoSearch([]);
+    } else {
+      setSearch(searchText);
+      setSelectedValue("");
+    }
+  };
+
+  useEffect(() => {
+    if (search === "") {
+      return;
```

Comment:
If you've already searched for something, then get rid of the search term, what will be displayed?